docker-images/base-stretch/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM debian:stretch
+MAINTAINER Will Kamp <manimaul@gmail.com>
+RUN rm -rf /var/lib/apt/lists/*
+RUN apt-get update
```

Comment:
You should be bundling these into a single RUN command. See [the best-practices for RUN](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run), which has a whole subsection on how to use apt-get properly in a Dockerfile.